pkg/api/scheme/scheme.go

```diff
@@ -69,10 +89,17 @@ var builders = []runtime.SchemeBuilder{
 }
 
 func init() {
-	Scheme = runtime.NewScheme()
-	if err := addKnownTypes(Scheme); err != nil {
+	Scheme = &FilteredScheme{
+		Scheme:       runtime.NewScheme(),
+		excludedGVKs: make(map[schema.GroupVersionKind]struct{}),
+	}
+
+	if err := addKnownTypes(Scheme.Scheme); err != nil {
 		panic(err)
 	}
+
+	// Node templates return values from node drivers that are not referenced on the `NodeTemplate`
+	Scheme.ExcludeGVK(management.SchemeGroupVersion.WithKind("NodeTemplateList"))
```

Comment:
Are you sure this is the only one? What about kev1 clusters? You can create a kev1 cluster so you can test this by going to this path `<rancher-url>/g/clusters/add/launch/amazoneks`.

Reply:
Alright, added `clusters.management.cattle.io` to the excluded GVKs and confirmed it addresses https://github.com/rancher/rancher/issues/35491. @rmweir , is that the same resource used by KEv1 or there is something else?

@paynejacob , is there any way you audit other types and see if we need to add more exclusions?

Reply:
I confirmed I can create and edit eks clusters with that endpoint you gave @rmweir

Reply:
@snasovich oh I think I misunderstood the question earlier, but yes what I'm speaking of is contained in clusters.management.cattle.io. @paynejacob great! Have you checked cloud credentials?

Reply:
cloud credentials are also good